docs/3.0.0-beta.x/concepts/middlewares.md

```diff
@@ -100,7 +65,49 @@ The framework allows the application to override the default middlewares and add
 
 Every middleware will be injected into the Koa stack. To manage the load order, please refer to the [Middleware order section](#load-order).
 
-## Load order
+## Configuration and activation
+
+To activate and configure your hook with custom options, you need to edit your `./config/environments/**/middleware.json` file in your Strapi app.
```

Comment:
Add a note about creating this file if it doesn't exist (which it doesn't by default)